<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemoryFormatDialog</class>
 <widget class="QDialog" name="MemoryFormatDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="message4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>AND &quot;ENTER&quot; TO FORMAT MEMORY</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="message1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>681</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>YOU ARE ABOUT TO FORMAT THE MEMORY!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="message5">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>190</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>PRESS ANY OTHER KEY TO CANCEL</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="message2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ALL SURVEY DATA WILL BE LOST</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="message3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>PRESS &quot;DEL&quot; TO COMFIRM</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="message6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>240</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Press DEL then ENTER to confirm</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
